components/buttons.css

```
@@ -0,0 +1,79 @@
+.buttons {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+}
+
+.buttons__100,
+.buttons__75,
+.buttons__50,
+.buttons__25 {
```

I agree with Annekes comment, the different background-colors would be modifications of this button. 
```suggestion
.energy-rating__button {
```